src/jimgw/samplers/blackjax/smc.py

```
@@ -0,0 +1,855 @@
+                    accept_list = list(_ckpt["accept_history"])
+                    cov_scale_list = list(_ckpt["cov_scale_history"])
+                    self._prev_elapsed = float(_ckpt["elapsed_time"])
+                    logger.info(
+                        "SMC-AP: resumed from checkpoint at n_iter=%d (%s)",
+                        n_iter,
+                        ckpt_path,
+                    )
+                else:
+                    state = smc_alg.init(initial_particles)  # type: ignore[call-arg]  # blackjax fork API
+                    self._prev_elapsed = 0.0
+            except Exception as _e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)